doc/contrib/documentation.md

```diff
@@ -0,0 +1,84 @@
+# Documentation
+
+This document gives guidelines for contributors concerning Elektra's documentation.
+This document takes preference to the actual situation, so if you see documentation
+not according to this document, please [create an issue](https://issue.libelektra.org)
+or directly fix it with your next PR.
+
+> Note: It is always allowed to improve the documentation, in every PR, even if
+> the documentation fix is completely unrelated.
```

Comment:
Maybe mention that a separate PR should still be prefer over adding a docs fix to a large and/or long-lived PR to ensure it is merged faster.